scripts/build/travis/configure_travis_trusty.sh

```diff
@@ -112,6 +112,7 @@ CMAKE_LIBS=(
 # List of applications to compile. Set to ON/OFF
 
 CMAKE_APPLICATION=(
+  -DMPI_NEEDED=ON
```

Comment:
That flag should be in `CMAKE_BUILD` section to avoid confusion (Line 135)

Reply:
makes sense